packages/lesswrong/server/resolvers/revisionResolvers.ts

```diff
@@ -36,10 +36,11 @@ augmentFieldsDict(Revisions, {
     type: String,
     resolveAs: {
       type: 'String',
-      resolver: ({originalContents}) => originalContents
+      resolver: ({originalContents}): string | null => originalContents
         ? dataToMarkdown(originalContents.data, originalContents.type)
         : null,
-    }
+    },
+    nullable: true,
```

Comment:
This is just filling in something that was already implicitly nullable?

Reply:
yeah I wasn't sure whether the default was true or false and this is definitely nullable